cocotb/share/makefiles/Makefile.sim

```diff
@@ -82,14 +82,26 @@ ifeq ($(MAKECMDGOALS),help)
     $(error "Stopping after printing help")
 endif
 
+# Safety net to bail out early if users go into this Makefile without making
+# cocotb-config available in their PATH.
+# Use override to prevent users from overriding COCOTB_CONFIG_CMD, which is not
+# supported. (Modify PATH instead.)
+override COCOTB_CONFIG_CMD := $(shell :; command -v cocotb-config)
+ifeq ($(COCOTB_CONFIG_CMD),)
```

Comment:
```suggestion
ifeq ($(shell :; command -v cocotb-config),)
```
I don't know if this would work, but it would also solve the nit Eric had at the bottom of this file.